weblate_web/templates/features.html

```diff
@@ -81,7 +81,7 @@ <h3>{% trans "Branch Support" %}</h3>
                 <a href="https://docs.weblate.org/en/latest/admin/translating.html#additional">
                     <div class="icon"><img src="{% static "img/icon-context.svg" %}" /></div>
                     <h3>{% trans "Translation Context" %}</h3>
-                    <p>{% trans "Context, comments, nearby translations, images and corresponding source code for every string." %}</p>
+                    <p>{% trans "Description, nearby translations, images, related source code and dictionary entries—for every string." %}</p>
```

Comment:
```suggestion
                    <p>{% trans "Description, images, age, related source code and dictionary entries—for every string." %}</p>
```
Nearby strings is not really in the context field. Maybe it fits into another category to not get that confused?

Reply:
```suggestion
                    <p>{% trans "Description, nearby strings, images, related source code and glossary entries—for every string." %}</p>
```